Sources/MuxPlayerSwift/FairPlay/FairPlaySessionManager.swift

```diff
@@ -158,41 +135,69 @@ class DefaultFairPlayStreamingSessionManager<
         drmToken: String,
         completion requestCompletion: @escaping (Result<Data, Error>) -> Void
     ) {
-        let url = URL(
+        guard let url = URLComponents(
             playbackID: playbackID,
             drmToken: drmToken,
-            applicationCertificateLicenseDomain: String.makeLicenseDomain(
-                rootDomain: rootDomain
+            applicationCertificateHostSuffix: rootDomain
+        ).url else {
+            logger.debug(
+                "Invalid FairPlay certificate domain \(rootDomain, privacy: .auto(mask: .hash))"
             )
-        )
+            requestCompletion(
```

Comment:
Previously this was an error that'd be triggered by a failing force unwrap (`!`) in the URL constructor.